core/cas-server-core-webflow-api/src/test/java/org/apereo/cas/web/flow/DefaultSingleSignOnBuildingStrategyTests.java

```diff
@@ -0,0 +1,58 @@
+package org.apereo.cas.web.flow;
+
+import lombok.val;
+import org.apereo.cas.support.events.logout.CasRequestSingleLogoutEvent;
+import org.apereo.cas.support.events.ticket.CasTicketGrantingTicketDestroyedEvent;
+import org.apereo.cas.ticket.TicketGrantingTicket;
+import org.apereo.cas.ticket.registry.TicketRegistry;
+import org.apereo.cas.ticket.registry.TicketRegistrySupport;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * This is {@link DefaultSingleSignOnBuildingStrategyTests}.
+ *
+ * @author Brian Kerr
+ * @since 7.2.0
+ */
+@Tag("Webflow")
```

Comment:
Your tests have nothing to do with the webflow. Probably should switch to using the Authentication tag instead.